engine/src/main/java/com/google/android/fhir/db/impl/entities/DateIndexEntity.kt

```diff
@@ -47,5 +45,5 @@ internal data class DateIndexEntity(
   @PrimaryKey(autoGenerate = true) val id: Long,
   val resourceType: ResourceType,
   @Embedded(prefix = "index_") val index: DateIndex,
-  val resourceId: String
+  val resourceLocalId: String,
```

Comment:
```suggestion
  val resourceUuid: String,
```

and all the other tables?